src/test/e2e/api/admin/feature.e2e.test.ts

```diff
@@ -324,9 +324,9 @@ test('refuses to create a new feature toggle with variant when type is json but
         .set('Content-Type', 'application/json')
         .expect(400)
         .expect((res) => {
-            expect(res.body.isJoi).toBe(true);
-            expect(res.body.details[0].type).toBe('invalidJsonString');
-            expect(res.body.details[0].message).toBe(
+            console.log(res.body);
+            expect(res.body.errors[0].type).toBe('invalidJsonString');
```

Comment:
Not actually sure we should have a `type` property on errors like this, but I'm also not against it. This isn't defined on the error type just yet, so maybe we can postpone that conversation?